lib/modules/datasource/rubygems/http.ts

```diff
@@ -0,0 +1,16 @@
+import { Http } from '../../../util/http';
+import { Throttle } from '../../../util/http/throttle';
+import { parseUrl } from '../../../util/url';
+
+export class RubygemsHttp extends Http {
+  protected override getThrottle(url: string): Throttle | null {
+    const host = parseUrl(url)?.host;
+
+    if (host === 'rubygems.org') {
+      const intervalMs = 100;
```

Comment:
```suggestion
      // rubygems.org has a rate limit of 15 per second, so we use a more conservative 10
      const intervalMs = 100;
```

Reply:
Actually, I think we're not conservative:
- API and website: 10 requests per second
- Dependency API: 15 requests per second

Reply:
(dependency API is the deprecated one, so we're referring to first one)